chapter-12-platform-specific-apps/desktop-app/src/renderer/callApi/callApiView.tsx

```diff
@@ -31,7 +31,7 @@ export function CallApiView(props: CallApiProps) {
 
         try {
             setErrorText('');
-            const orders = await apiRequest<Order[]>(IpcEventNames.ApiOrders);
+            const orders = await makeApiRequest({name: IpcEventNames.ApiOrders}) as Order[];
```

Comment:
Would it make sense to introduce two functions in the apiRequest file: `getOrders` and `getUserInfo`? These could then call the generic method. This should then simplify the code:

```
const orders = await getOrders();
```

```
async function getOrders(): Promise<Order[]> {
    return await makeApiRequest({name: IpcEventNames.ApiOrders}) as Order[];
}
```

Reply:
Good idea - updated